bin/start-chronos.bash

```diff
@@ -1,27 +1,39 @@
 #!/bin/bash
-set -o errexit -o nounset -o pipefail
+set -o errexit -o pipefail
 
-chronos_home="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd -P )"
-echo "Chronos home set to $chronos_home"
-export JAVA_LIBRARY_PATH="/usr/local/lib:/lib:/usr/lib"
-export LD_LIBRARY_PATH="${LD_LIBRARY_PATH:-/lib}"
-export LD_LIBRARY_PATH="$JAVA_LIBRARY_PATH:$LD_LIBRARY_PATH"
```

Comment:
I don't see how it would make any difference @elingg, if you mean `if [ -z "$1" ]...` statement.
The _unset_, as I understand causes script to exit with error if any variable used is unset.. So what would it be catching there after if statement?